### PR TITLE
Create CreateNewRepository.md

### PR DESCRIPTION
**Pull request by MarkusKuemmerle**

### CODE REVIEW

File: documentation/CreateNewRepository.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	* Insert all maintainers
+* Update README.MD
+	* Replace all occurances of §...§ adequately
+	* Define the value for §api_family_abbreviation§
+
+### Mailing List
+
+* Create Mailing Subgroup:
+	* Choose "Create Subgroup" in https://lists.camaraproject.org/g/main/createsubgroup
+	* Insert value for §api_family_abbreviation§ and a description (following the pattern of the existing Subgroups)
```
Comment (hdamker) [inline documentation/CreateNewRepository.md:36]:
```suggestion
	* Insert value for "sp-§api_family_abbreviation§" and a description (following the pattern of the existing Subgroups)
```

File: documentation/CreateNewRepository.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Add branch rules:
+  * Select "Settings" --> "Branches"
+  * Select button "Add branch protection rule"
+  * Insert "main" as pattern name
+  * Select option "Require a pull request before merging"
+  * Select option "Require approvals"
+  * Select option "Dismiss stale pull request reviews"
+  * Select option "Require review from Code Owners"
+  * Select button "Create"
+  * Select button "Add rule" and repeat it with pattern "*release*"
```
Comment (hdamker) [inline documentation/CreateNewRepository.md:19]:
```suggestion
  * Select button "Add rule" and repeat it with pattern "\*release\*"
```

**Review by hdamker — COMMENTED** [review]
See two necessary corrections. Beyond that I recommend to move the file to "SupportingDocuuments" (just edit the file name for that).

**Review by hdamker — CHANGES_REQUESTED** [review]
We discussed in admin team to move the content into [wiki.camaraproject.org](https://wiki.camaraproject.org) => don't merge here.

@wrathwolf can you please confirm when this is done so we can close the PR here?

**Comment by MarkusKuemmerle** [comment]
Content is moved into the Wiki

**Pull request closed without merging**